### ISSUE TITLE
The given groupId in documentation is wrong

### ISSUE THREAD

**Issue by khmarbaise** [buschmais/jqassistant#391]
In the docs you will find: 
4.2.2. Add the plugin
and there the entry:
```xml
<plugin>
            <groupId>com.buschmais.jqassistant.scm</groupId>
            <artifactId>jqassistant-maven-plugin</artifactId>
            <executions>
```
which shows wrong groupId. This should be changed to the current groupId.

**Comment by DirkMahler** [comment]
Has already been fixed for 1.3.0 - sorry for the inconvenience...

**Comment by khmarbaise** [comment]
No problem...really no need for excuse...Thanks for fixing it...
